src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -665,7 +665,7 @@ protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
 
                 var x = Offset.X;
                 var y = Offset.Y;
-                var delta = e.Delta;
+                var delta = CheckFlowDirection(e.Delta);
```

Comment:
This should be done right after the `if` block below, so that `Shift+Wheel Up` correctly scrolls left and `Shift+Wheel Down` correctly scrolls right in RTL mode.